ngraph/frontend/onnx_import/src/op/org.openvinotoolkit/group_norm.cpp

```diff
@@ -38,9 +38,11 @@ namespace ngraph
                         auto splits = builder::opset1::split(shape, rank_size);
                         auto num_groups_const =
                             default_opset::Constant::create(element::i64, Shape{1}, {num_groups});
+                        // The 4D shape: [N * num_groups, C // num_groups, H, W] is created
+                        // instead of 5D shape: [N, num_groups, C // num_groups, H, W].
+                        // The reason is the lack of support for 5D MVN input by some plugins.
                         ngraph::OutputVector new_shape{
-                            splits[0],
-                            num_groups_const,
+                            std::make_shared<default_opset::Multiply>(splits[0], num_groups_const),
```

Comment:
Do we need to add new tests? Or is it covered by some existed tests?

Reply:
@ilyachur This change has no effect on the results, so in my opinion the current tests are sufficient.

We are not able (with the current unit test infrastructure) to create a test that will start passing (on tested backends) after that commit.

The following unit tests cover this line:
onnx_group_norm
onnx_group_norm_5d
onnx_model_experimental_detectron_group_norm

Reply:
Thank you for information.